test/servico/cst06Test.java

```diff
@@ -68,10 +73,13 @@ public static void tearDownClass() {
     @Before
     public void setUp() throws ParseException {
         ProfessorService professorService = new ProfessorService();
-        ProfessorModel professor = professorService.obterProfessorPorNome("Professor2");
+        ProfessorModel professor = 
+                professorService.obterProfessorPorNome("Professor2");
         
         NotificacaoService notificaçãoService = new NotificacaoService();
-        notificaçãoService.notificarAusencia(professor.id, "25/11/2013 18:30", "25/11/2013 19:59", "Motivo Declarado", new LinkedList<String>());
+        notificaçãoService.notificarAusencia(
+                professor.id, "25/11/2013 18:30", "25/11/2013 19:59", 

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The String "25/11/2013 18:30" appears 2 times in the file.](https://www.codacy.com/app/calebepb/substituicao/file/2777729603/issues/source?bid=3364462&fileBranchId=3364484#l81)